client/src/components/Creator.js

```diff
@@ -18,8 +18,24 @@ class Creator extends Component {
   }
 
   async getBalance() {
-    const tokenAddress = "0xbcca60bb61934080951369a648fb03df4f96263c";
+    const tokenAddress = "0xe12afec5aa12cf614678f9bfeeb98ca9bb95b5b0";
     const walletAddress = this.props.connectedWallet;
+    const fetchURL =
+      "https://api-kovan.etherscan.io/api?module=account&action=tokenbalance" +
+      "&contractaddress=" +
+      tokenAddress +
+      "&address=" +
+      walletAddress +
+      "&tag=latest&apikey=" +
+      process.env.ETHERSCAN_KEY;
+
+    fetch(fetchURL)
+      .then((response) => response.json())
+      .then((balance) => {
+        balance = (balance.result * Math.pow(10, -6)).toFixed(2);
```

Comment:
Nit: Could you add a comment explaining what conversion the math does?

Reply:
Absolutely.